README.md

```diff
@@ -10,12 +10,10 @@ cmake -S . -B build -A Win32
 cmake --build build
 ```
 
-## xi_checker
+## Release notes
 
-Verifies DirectPlay and POL are installed and connect to a server using xiloader (installed in POL folder).
+#Windows:
```

Comment:
Space between the # and the content for titles, no need for the :

Reply:
Done